extensions/data-workspace/src/dataworkspace.d.ts

```diff
@@ -75,6 +76,11 @@ declare module 'dataworkspace' {
 		 * Gets the project actions to be placed on the dashboard toolbar
 		 */
 		readonly projectActions: (IProjectAction | IProjectActionGroup)[];
+
+		/**
+		 * Gets the project data to be placed in the dashboard container
+		 */
+		readonly dashboardData: IDashboardTable[];
```

Comment:
how about dashboardComponents, in the right now we only support table, we can have other component types in the future.